composer.json

```diff
@@ -56,6 +56,9 @@
         "composer/bin/phpunit"
     ],
     "autoload": {
+    	"psr-4": {
+			"PHPUnit\\": "PHPUnit"
+		},
```

Comment:
indent consistency issue: previously space indent was used not tabs, also seems out of place (see it in github browser)

Reply:
@glensc Thank you!